tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -4,7 +4,6 @@
 # SPDX-License-Identifier: FSFAP
 #
 # Summary: For openSUSE virtualization test only. login in console, install kvm/xen patterns if needed.
-#  - Even if you'd like to run tests without host installation(DO_NOT_INSTALL_HOST=1), this module is still necessary as login console in this module is required.
 #  - This module is added for openSUSE TW because of the difference beteen SLE and TW. Meanwile, login_console, install_package and update_package from SLE are not needed. The reasons are listed below:
```

Comment:
Hi Julie, would you please add a comment in this file summary to let readers know it is not stable to use this file for TW test and the short main reasons?

Reply:
Yes, done locally. will push in a moment.